extensions/amp-image-lightbox/0.1/amp-image-lightbox.js

```diff
@@ -706,6 +706,11 @@ class AmpImageLightbox extends AMP.BaseElement {
     this.reset_();
     this.init_(source);
 
+    /**  @private {function(this:AmpLightbox, Event)}*/
```

Comment:
Either `AmpImageLightbox` or you can simply do `{!Function}` - the exact signature in this case doesn't matter - you never call it in the code.